lib/parser.js

```diff
@@ -130,10 +130,17 @@ function scope (scanner) {
     node.value += scanner.next()
   }
 
+  if (scanner.peek() !== ')') {
+    throw scanner.abort(node, [')'])
```

Comment:
Note, this is an important `throw`. If there is no closing tag, it's always EOF and should abort immediately. We currently have these concepts when parsing tokens:
- Token is fully parsed, here is the node
- Token failed to parse, this is why

But, if we don't want to throw right away, we have to add another concept:
- Token failed to parse, but its still ok, the token is optional

We need to differentiate between a "permissive"/optional error, or a fatal one like no-closing `)`.